tests/Jobs/ElasticSearch/ElasticSearchIndexDeleteTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Tests\Jobs;
+namespace Tests\Jobs\ElasticSearch;
```

Comment:
This might be incorrect if the namespace changes to CirrusSearch, this job actually interacts directly with ElasticSearch.

Reply:
Perhaps this one should remain in ElasticSearch then?

Reply:
yup